packages/gguf/src/gguf.ts

```diff
@@ -3,7 +3,7 @@ import { GGUFValueType } from "./types";
 import { promisesQueue } from "./utils/promisesQueue";
 
 export type { MetadataBaseValue, MetadataValue, Version, GGUFMetadata, GGUFTensorInfo, GGUFParseOutput } from "./types";
-export { GGUFValueType, GGMLQuantizationType } from "./types";
+export { GGUFValueType, GGMLQuantizationType, GGUFArchitecture } from "./types";
```

Comment:
I'm wondering if it should be named `GGMLArchitecture`, because gguf is more like a container, so it does not contains the actual architecture. It's a bit like `GGMLQuantizationType` is defined at tensor level (ggml) and rather than at gguf level.

(FYI, before gguf, llama.cpp used `*.ggml` format to distribute tensor data)

Reply:
ok i'll revert that second commit then